src/Microsoft.ML.AutoML/Experiment/Runners/RunnerUtil.cs

```diff
@@ -10,6 +10,8 @@ namespace Microsoft.ML.AutoML
 {
     internal static class RunnerUtil
     {
+        private static ITransformer _estimatorModel;
```

Comment:
Why does this need to be a class variable? Previously the estimator and model were local to the function. Can it not remain local to the function?